automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/analysis/MethodConfigSpec.scala

```diff
@@ -27,11 +27,12 @@ class MethodConfigSpec extends FreeSpec with WebBrowserSpec with CleanUp with Wo
     val user = Config.Users.owner
     implicit val authToken: AuthToken = user.makeAuthToken()
     withWorkspace(billingProject, "TestSpec_FireCloud_launch_a_simple_workflow") { workspaceName =>
+      api.workspaces.waitForBucketReadAccess(billingProject, workspaceName)
```

Comment:
I moved some of these to before `withSignIn` so avoid prematurely starting the clock on selenium killing the browser session due to inactivity.